articles/aks/keda-deploy-add-on-cli.md

```diff
@@ -0,0 +1,537 @@
+                  type: object
+                type: array
+              externalMetricNames:
+                items:
+                  type: string
+                type: array
+              health:
+                additionalProperties:
+                  description: HealthStatus is the status for a ScaledObject's health
+                  properties:
+                    numberOfFailures:
+                      format: int32
+                      type: integer
+                    status:
+                      description: HealthStatusType is an indication of whether the
+                        health status is happy or failing
+                      type: string
+                  type: object
+                type: object
+              lastActiveTime:
+                format: date-time
+                type: string
+              originalReplicaCount:
+                format: int32
+                type: integer
+              pausedReplicaCount:
+                format: int32
+                type: integer
+              resourceMetricNames:
+                items:
+                  type: string
+                type: array
+              scaleTargetGVKR:
+                description: GroupVersionKindResource provides unified structure for
+                  schema.GroupVersionKind and Resource
+                properties:
+                  group:
+                    type: string
+                  kind:
+                    type: string
+                  resource:
+                    type: string
+                  version:
+                    type: string
+                required:
+                - group
+                - kind
+                - resource
+                - version
+                type: object
+              scaleTargetKind:
+                type: string
+            type: object
+        required:
+        - spec
+        type: object
+    served: true
+    storage: true
+    subresources:
+      status: {}
+    status: "True"
+    type: NamesAccepted
+  - lastTransitionTime: "2022-06-08T10:31:07Z"
+    message: the initial names have been accepted
+    reason: InitialNamesAccepted
+    status: "True"
+    type: Established
+  storedVersions:
+  - v1alpha1
+  ```
+
+KEDA provides multiple customization options. The KEDA add-on enables the basic common configuration. If you have requirement to run with some  custom configurations for example:  number of KEDA pods running, namespaces that should be watched, logging level, etc. then you may edit the KEDA yaml manually and deploy it. Remember the customizations would be managed by you. 
+
+## Disable KEDA add-on from your AKS cluster
+
+When you no longer need KEDA add-on in the cluster, use the `az aks update` command with--disable-keda option. This will disable KEDA workload auto-scaler
+
+```azurecli-interactive
+az aks update \
+  --resource-group myResourceGroup \
+  --name myAKSCluster \
+  --disable-keda 
+```
+
+## Troubleshooting guide
```

Comment:
Added to ARM page as well. Since we have only one error it didnt make sense to create a page for this. As we progress we can add further potential known issues